libpod/container_inspect.go

```diff
@@ -192,14 +192,9 @@ func (c *Container) getContainerInspectData(size bool, driverData *driver.Data)
 // Get inspect-formatted mounts list.
 // Only includes user-specified mounts. Only includes bind mounts and named
 // volumes, not tmpfs volumes.
-func (c *Container) getInspectMounts(namedVolumes []*ContainerNamedVolume, mounts []spec.Mount) ([]define.InspectMount, error) {
+func (c *Container) getInspectMounts(namedVolumes []*ContainerNamedVolume, imageVolumes []*ContainerImageVolume, mounts []spec.Mount) ([]define.InspectMount, error) {
 	inspectMounts := []define.InspectMount{}
 
-	// No mounts, return early
-	if len(c.config.UserVolumes) == 0 {
```

Comment:
We need to retain this - if it's no longer accurate because image volumes/overlay volumes aren't being added to it, they should be added to UserVolumes.